cpp/src/arrow/util/bitmap_writer.h

```diff
@@ -78,96 +79,24 @@ class BitmapWriter {
   int64_t byte_offset_;
 };
 
-class FirstTimeBitmapWriter {
+class ARROW_EXPORT FirstTimeBitmapWriter {
```

Comment:
Since the other classes are fully defined in the header you should not have to.  Technically it wouldn't hurt either but best to be minimal with changes probably.

Reply:
Correct.